packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -2595,10 +2595,22 @@ abstract class Element extends DiagnosticableTree implements BuildContext {
   /// only be called if it is provable that the children are available.
   void visitChildren(ElementVisitor visitor) { }
 
-  /// Calls the argument for each child that is relevant for semantics. By
-  /// default, defers to [visitChildren]. Classes like [Offstage] override this
-  /// to hide their children.
-  void visitChildrenForSemantics(ElementVisitor visitor) => visitChildren(visitor);
+
+  /// Calls the argument for each child that is onstage.
+  ///
+  /// An element is considered to be onstage when a user is expected to be able
+  /// to interact with it. The precise definition of "interaction" is specific
+  /// to the widget, and may include hit testing, semantics, visibility as well
+  /// as other properties.
+  ///
+  /// The default implementation defers to [visitChildren] and therefore treats
+  /// the element as onstage.
+  ///
+  /// See also:
+  ///
+  /// - [Offstage]
```

Comment:
nit: I often find it helpful to have a sentence fragment here explaining why I should also look at Offstage and visitChildrenForSemantics.